src/fu-main.c

```diff
@@ -2368,23 +2369,32 @@ main(int argc, char *argv[])
 	}
 #endif
 
-	/* own the object */
+	/* convert from filename to address, if required */
```

Comment:
similar to my other comment - I think hardcode the daemon to listen on TCP for Windows here.